src/azul/__init__.py

```diff
@@ -677,7 +677,10 @@ def is_dss_enabled(self, catalog: Optional[str] = None) -> bool:
         return self._is_plugin_enabled('dss', 'repository', catalog)
 
     def is_tdr_enabled(self, catalog: Optional[str] = None) -> bool:
-        return self._is_plugin_enabled('tdr_hca', 'repository', catalog)
+        return any(
+            self._is_plugin_enabled(plugin, 'repository', catalog)
+            for plugin in ['tdr_hca', 'tdr_anvil']
```

Comment:
It would be simpler and less brittle if we used a prefix comparison. I would split the plugin name on `_` and return `True` if the first component of any plugin name, is `tdr`, regardless of type. Analogously, for `dss` and `hca`.